packages/wdio-applitools-service/applitools-service.d.ts

```diff
@@ -19,17 +20,17 @@ interface ApplitoolsConfig {
      * Viewport with which the screenshots should be taken.
      */
     viewport?: {
-        width: number;
-        height: number;
+        width?: number;
+        height?: number;
     };
     /**
      * Use proxy for http/https connections with Applitools.
      */
     proxy?: {
         url: string;
-        username?: string;
-        password?: string;
-        isHttpOnly?: boolean;
+        username: string;
+        password: string;
+        isHttpOnly: boolean;
```

Comment:
should all of them be really required?

Reply:
Here's a definition of ProxySettings in applitools - https://github.com/applitools/eyes.sdk.javascript1/blob/master/packages/eyes-sdk-core/lib/config/Configuration.js#L346-L362. I'm not sure if it's a correct version of SDK webdriverio uses, but it looks like, when an object is passed, all properties are required.